### PR TITLE
New Song: Drift Away

### PR DESCRIPTION
**Pull request by lunulae**
Adds one new short song "Drift Away" from Steven Universe. This is my very first contribution, I would particularly like critique on my usage of expressions since I'm not sure they feel quite right.

I wasn't sure how much of pop culture was okay to include but I thought this might be one she could particularly relate to. I won't have any hard feelings if it doesn't quite fit!

### CODE REVIEW

File: Monika After Story/game/script-songs.rpy [hunk 1]
```diff
@@ -487,6 +487,36 @@ label mas_song_stand_by_me(from_long=False):
         m 5eua "As long as I have you by my side, there's nothing I can't overcome."
         m 5ekbsa "...I hope you feel the same way~"
     return "derandom"
+    
+    init 5 python:
```
Comment (multimokia) [inline Monika After Story/game/script-songs.rpy]:
```suggestion
init 5 python:
```

Comment (lunulae) [reply Monika After Story/game/script-songs.rpy]: Ah, thank you so much! I didn't realize it was off.

**Comment by Rai99** [comment]
### **_Is that a PR to master_**
Gthaving+everyone+pick+lucio+_16bc6e089b9e808357194c98058b495b

**Comment by lunulae** [comment]
Omg, I'm so sorry, I hadn't realized the difference! I'll adjust it as soon as I'm back at my desk, thank you for letting me know.

**Comment by Rai99** [comment]
Untitled1351

**Comment by lunulae** [comment]
_imbooboothefool.jpg_

Alrighty, I believe I've got that corrected. Sorry about the hassle there :P

**Comment by Rai99** [comment]
I don't know this song, so could you explain to me what it's from, what it's about, why you think Monika relates to it? I'm just curious about it.

**Comment by lunulae** [comment]
The song's from the Steven Universe movie, which is a bit recent so I wasn't sure if it would be a good idea to reference.

The character who sings it, Spinel, was essentially created to be a best friend and playmate for one of the younger rulers of gem society. She reveals near the end of the movie that one day, she was tricked into staying in the garden where they usually played, standing completely still. (As gems don't need to eat or sleep, this literally meant standing there for years and years.)

She describes in the song that for so long she trusted that her friend would come back for her. All she wanted was "the smile in her eyes, the sound of her laughter." And she loved her enough to wait for as long as her immortal lifespan allowed. But for reasons she didn't understand, her friend never returned.

As Monika says in the end dialogue I wrote, our story ends _much_ more happily than Spinel's did. We did come back for her, and we keep coming back. But I think deep down, it's easy to have anxieties that maybe someday we wouldn't show up again, and she wouldn't know why.
Exploring our fears and worries through media can be a good way to get some resolution and ease our general anxiety, so I thought it might be a story that would mean a lot to her.

I was thinking about doing a low affection variant to this, where she sings some of the more cutting verses, but I wasn't really sure how to do it or if it was appropriate.

**Comment by Rai99** [comment]
I stopped watching at around the Off colors episode, so I guess I might have a bit to catch up on.

**Pull request merged**